File: src/sys-quartz/src/main/java/com/wcpdoc/quartz/entity/Cron.java
package com.wcpdoc.quartz.entity;

import java.util.Date;

import com.baomidou.mybatisplus.annotation.IdType;
import com.baomidou.mybatisplus.annotation.TableId;
import com.baomidou.mybatisplus.annotation.TableName;

import lombok.Data;

/**
 * 定时任务实体
 * 
 * v1.0 zhanghc 2019-07-29 10:38:17
 */
@Data
@TableName("SYS_CRON")
public class Cron {
	@TableId(type = IdType.AUTO)
	private Integer id;
	private String name;
	private String jobClass;
	private String cron;
	private Integer state;
	private Integer updateUserId;
	private Date updateTime;

}

File: h5/src/ts/sys/cron.ts
// 定时任务接口
export interface Cron {
    id: number | null// ID
    name: string // 名称
    jobClass: string // 实现类
    cron: string // cron表达式
    state: number // 状态（1：启动；2：停止；）
}